### ISSUE TITLE
Balance just changed suddenly due to funds pending

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#109]
https://rumblefishdev.atlassian.net/browse/ZBAY-861

**Comment by NorbertBodziony** [comment]
Unfortunately zcashd does not provide accurate way to calculate pending amount. I don't think there is a way we can do it internally. Maybe we should push request to Zcash Team ? @holmesworcester

**Comment by holmesworcester** [comment]
@NorbertBodziony yes, let's do that. can you create the issue and ping me when it has been created? thanks!

**Comment by NorbertBodziony** [comment]
@holmesworcester issue created https://github.com/zcash/zcash/issues/4586.
